packages/ui/src/memberships/components/MemberListItem/MemberListItem.tsx

```diff
@@ -23,43 +25,77 @@ export const MemberListItem = ({ member }: Props) => {
   }, [member.id])
 
   return (
-    <MemberItemWrap>
+    <MemberItemWrap member={member}>
+      {member.type === 'Members' && (
+        <MemberColumn>
+          <Info>#{member.id}</Info>
+        </MemberColumn>
+      )}
+
       <MemberColumn>
-        <MemberInfo member={member} onClick={showMemberModal} showId />
+        <MemberInfo member={member} onClick={showMemberModal} showId={member.type !== 'Members'} />
       </MemberColumn>
+
+      {member.type === 'Members' && (
+        <MemberColumn>
+          <Info>{member.isConcilMember ? 'YES' : 'NO'}</Info>
+        </MemberColumn>
+      )}
```

Comment:
Introducing multiple type checks may indicate that this component might be a new one. And, I think that this should be true also here. The reason is that it starts getting complicated to support different types in one component.

I think that it would be better to revert the changes made here and introduce a new component for memberships lists.

To help you with naming, as you might not be accustomed to the projects domain:

*   the existing one can be renamed to `<MyMembers />`  as it is used to displays user's memberships (members)
*   the new one can be named simply `<Members />` as it displays a list of all of the Pioneer's users